draft-ietf-quic-recovery.md

```diff
@@ -389,6 +398,13 @@ rttvar_sample = abs(smoothed_rtt - adjusted_rtt)
 rttvar = 3/4 * rttvar + 1/4 * rttvar_sample
 ~~~
 
+An endpoint might postpone the processing of acknowledgements when the
+corresponding decryption keys are not immediately available. For example, a
+client might receive an acknowledgement for a 0-RTT packet that it cannot
+decrypt because 1-RTT packet protection keys are not yet available to it. In
+such cases, an endpoint SHOULD ignore such local delays in its round-trip time
+sample until the handshake is confirmed.
```

Comment:
```suggestion
An endpoint might receive packets in ApplicationData containing
acknowledgements which cannot be processed immediately due to a
lack of 1-RTT decryption keys. For example, a client might receive an
acknowledgement for a 0-RTT packet that it cannot decrypt because
1-RTT packet protection keys are not yet available to it. In such cases,
an endpoint SHOULD subtract the time the packet was buffered and
unable to be decrypted from the round-trip time sample.
```